src/app/pages/instances/components/instance-wizard/instance-wizard.component.html

```diff
@@ -143,6 +143,15 @@
       </ix-form-section>
     }
 
+    <ix-form-section [label]="'Storage' | translate">
```

Comment:
Depending on what this field does, it can be part of Disks section.
Alternatively rename Disks to Storage.